libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/TripSessionState.kt

```diff
@@ -3,7 +3,7 @@ package com.mapbox.navigation.core.trip.session
 /**
  * Describes the [TripSession]'s state.
  */
-enum class TripSessionState {
```

Comment:
nit, remove

Reply:
??

Reply:
Oh. Now I see it. Not sure how that extra space got there.